src/server/raftkv2/node.rs

```diff
@@ -85,6 +85,7 @@ where
         snap_mgr: TabletSnapManager,
         concurrency_manager: ConcurrencyManager,
         causal_ts_provider: Option<Arc<CausalTsProviderImpl>>, // used for rawkv apiv2
+        lock_manager_observer: Arc<dyn LockManagerNotifier>,
```

Comment:
Use Option.

Reply:
Why, we can use a dummy notifier if it is not needed (in test for example).

Reply:
Clearly None is better than dummy?

Reply:
Anyway, if previously coprocessor host is not an option, I'm okay with dummy too.